src/content/troubleshooting/_index.en.md

```diff
@@ -0,0 +1,135 @@
++++
+title = "Troubleshooting Guide"
+date = 2020-05-04T19:01:14+05:30
+weight = 5
+pre = "<b>1. </b>"
++++
+
+## Overview
+
+You have followed steps in [Deployment](../deployment) but something has gone wrong. You're not sure what and how to fix it, or what information to collect to raise an issue. Welcome to the Submariner troubleshooting guide where we will help you get your deployment working again.
+
+To recap, Submariner consists of two main core components (broker and submariner) and optional components (lighthouse), detailed in [Architecture](../architecture) section. Basic familiarity with the architecture will be helpful when troubleshooting.
+
+The guide has been broken into different sections for easy navigation.
+
+### Pre-requisite
+Before we begin troubleshooting, it would be good to know which version of submariner and its components you are running.
+
+Run `subctl version` to get version information.
+
+Get information about the service you're trying to access.
+
+`kubectl get services |grep <service-name>` will provide you with Service *Name*, *Namespace* and *ServiceIP*. If you enabled **Globalnet** you will also need the *GlobalnetIP* of the service by running
+
+``` kubectl get service <service-name> -o jsonpath='{.metadata.annotations.submariner\.io/globalIp}' ```
+
+<!---
+### Deployment Issues
+This section will contain information about common deployment issues you can run into.
+
+#### TBD
+
+### Connectivity Issues
+Deployment went through successfully but services/pods on one cluster are unable to connect to services on another clusters. This can be due to multuple factors - IP Sec tunnels, IP Table rules, Gateway pods, Routeagent etc.
+
+#### IPSec tunnel not created between clusters
+TBD
+
+#### IPSEc tunnel is not up between clusters
+TBD
+
+#### None of pods/services able to connect to remote service
+TBD
+##### Without Globalnet
+TBD
+##### With Globalnet
+TBD
+
+#### Pods on non-gateway nodes not able to connect to remote service
+TBD
+##### Without Globalnet
+TBD
+##### With Globalnet
+TBD
+
+-->
+
+### Service Discovery Issues
+If you are able to connect to remote service by using ServiceIP or GlobalnetIP, but not by service name, it is a Service Discovery Issue.
+
+#### Service Discovery not working
+This is good time to familiarize yourself with [Lighthouse Architecture](../architecture/components/lighthouse) if you haven't already.
+
+##### Check CoreDNS Configuration
```

Comment:
Maybe a section to check operator CR too , if it is an operator based deployment?

Reply:
That would go in Deployment section of the guide.